docker-compose-develop.yml

```diff
@@ -3,19 +3,55 @@ services:
   develop:
     build: .
     ports:
-      - "4000:4000"
+      - "30521:30521"
     container_name: document
     env_file:
       - dev.env
     environment:
       NODE_ENV: dev
-      PORT: 4000
+      PORT: 30521
       NODE_PATH: app/src
       MICROSERVICE_TOKEN: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6Im1pY3Jvc2VydmljZSIsImNyZWF0ZWRBdCI6IjIwMTYtMDktMTQifQ.IRCIRm1nfIQTfda_Wb6Pg-341zhV8soAgzw7dd5HxxQ
-      STAMPERY_TOKEN:
-      S3_ACCESS_KEY_ID:
-      S3_SECRET_ACCESS_KEY:
-      ELASTIC_URL: http://localhost:9200
+      GATEWAY_URL: http://mymachine:9000
```

Comment:
same thing as I mentioned on a different PR: there are better ways to do this. This, as is,  will cause problems

Reply:
Simply adding a missing environment variable here the way you have set them in the other MSs (e.g., dataset) to get this working. Can we do a system-wide change in a separate issue to address your concern?

Reply:
the concern is not the variable, is the value.

Reply:
Understood, but that's the recommended approach in the docs (e.g., in [dataset](https://github.com/resource-watch/dataset/blob/dev/docker-compose-develop.yml#L13) you have the same value in docker-compose). I can take it out into the dev.env file that's not version controlled if this feels hard-coded.

Reply:
those instructions are old, and should be reviewed. AFAIK those are Mac/Windows specific, and as they currently stand, they are incomplete, as there are no docs explaining what that fake host is supposed to mean - without which they won't work. This is another reason why I would encourage you to focus these changes on the nodejs version upgrade, and leave everything else for separate PRs